src/contracts/storage/pool_key.rs

```diff
@@ -5,6 +5,8 @@ use crate::contracts::FeeTier;
 use crate::contracts::InvariantError;
 use crate::math::percentage::Percentage;
 
+pub const MAX_POOL_KEYS_RETURNED: u16 = 220;
```

Comment:
maybe something more dynamic would be better: `MAX_RESULT / POOL_KEY_SIZE` (like we have it in wasm)

Reply:
definitely